### PR TITLE
(fix) Form entry loading regression in fast data entry app

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Fixes a regression in the [fast-data-entry app](https://github.com/openmrs/openmrs-esm-fast-data-entry-app) where the form engine would fail to load a form at all if concepts are missing.

> Screenshots (console error from running fast-data-entry-app)

Screenshot 2022-12-16 at 21 11 24

### CODE REVIEW

**Review by ibacher — APPROVED** [review]

File: packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.ts [hunk 1]
```diff
@@ -90,13 +90,15 @@ export class FeWrapperComponent implements OnInit, OnDestroy {
       .subscribe(
         ({ form, concepts }) => {
           this.form = form;
-          this.labelMap = concepts.reduce((acc, current) => {
-            if (Boolean(current)) {
-              acc[current.identifier] = current.display;
-            }
+          if (concepts) {
+            this.labelMap = concepts?.reduce((acc, current) => {
```
Comment (ibacher) [inline packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.ts:94]:
```suggestion
            this.labelMap = concepts.reduce((acc, current) => {
```

Comment (denniskigen) [reply packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.ts:94]: Leads to this:

Screenshot 2022-12-16 at 22 05 18

Comment (ibacher) [reply packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.ts:94]: Even with the new `if` statement?

Comment (denniskigen) [reply packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.ts:94]: Oh sorry, that was without the if

**Pull request merged**